packages/sui-logger/src/server/expressMiddleware/DataDogLogger.js

```diff
@@ -28,7 +28,7 @@ export class DataDogLogger extends Writable {
       method: req ? req.method : '-' // eslint-disable-line
     }
 
-    if (!req || statusFamily === '4xx' || statusFamily === '3xx') {
```

Comment:
do we know the reason why these status code families were an exception in the first place...?

Reply:
As far as @carlosvillu  and @andresz1  told me, the reason should be the increase of tags.

Adding this PR we will increase the number of tags for the server.request metric as much as routerUrls we have in the project ( x2, one for 3xx and the other for 4xx) In fotocasa for example it means add 80 tags

Reply:
On the other hand we get the option to monitor increases/decreases (anomalies) of  3xx and 4xx.
For example, that days we cannot create a monitor to detect an increase  of 400x errors for a specific page. Only global